extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -690,6 +690,9 @@ export class AmpA4A extends AMP.BaseElement {
                     return bytes;
                   case VerificationStatus.UNVERIFIED:
                     return null;
+                  case VerificationStatus.CRYPTO_UNAVAILABLE:
+                    return this.preferentialRenderWithoutCrypto() ?
```

Comment:
Wait is this backward?  if true shouldn't this return true and not null?

Reply:
damn good catch. Sorry about that. Fixed.